### ISSUE TITLE
triple-slash directives should remain top of file

### ISSUE THREAD

**Issue by thomasf7** [SoominHan/import-sorter#28]
I am having an issue with import-sorter causing conflicts by moving the amd-module name directive from the top of the file.

For example:
```
/// <amd-module name="ExampleModuleName" />
import { effects } from "redux-saga";
import { LoadSuccess } from "ExampleActions";
```

This will sort and move the redux-saga import below ExampleActions, but will also treat the amd-module directive as a comment on redux-saga and move it too, resulting in:

```
import { LoadSuccess } from "ExampleActions";
/// <amd-module name="ExampleModuleName" />
import { effects } from "redux-saga";
```

And when the modulename is regenerated:
```
/// <amd-module name="ExampleModuleName" />
import { LoadSuccess } from "ExampleActions";
/// <amd-module name="ExampleModuleName" />
import { effects } from "redux-saga";
```

Since triple slash directives are only valid at the top of a file, can an exception be added to leave them in place when sorting imports.

**Comment by bendman** [comment]
This may be unrelated, but a similar issue occurs with [shebangs](https://en.wikipedia.org/wiki/Shebang_(Unix)) (e.g. `#!/usr/bin/env node`).

**Comment by mike-co** [comment]
Fixed in 3.3.0
